thirdeye-server/src/main/java/ai/startree/thirdeye/ThirdEyeServerModule.java

```diff
@@ -69,4 +87,62 @@ public OAuthConfiguration getOAuthConfig(
       AuthConfiguration authConfiguration) {
     return authConfiguration.getOAuthConfig();
   }
+
+  @Singleton
+  @Provides
+  public BasicAuthConfiguration getBasicAuthConfig(
+      AuthConfiguration authConfiguration) {
+    return authConfiguration.getBasicAuthConfig();
+  }
+
+  @Singleton
+  @Provides
+  public AuthFilter getAuthFilters(final OAuthManager oAuthManager) {
+    AuthConfiguration config = configuration.getAuthConfiguration();
+    List<AuthFilter> filters = new ArrayList<>();
+    if(config.isEnabled()) {
+      optional(getBasicAuthFilter()).ifPresent(filters::add);
```

Comment:
Let's make the decision to create them here instead of returning `null`

Reply:
Sure